Update 2.x docs for master to manager rename.

brianloss opened the pull request:
* Update server-properties.md with updated (renamed) properties.
* Update references to master under _docs-2 to manager (not including
  apidocs).
* Document binary/wire incompatibility between 2.0.x and 2.1.
* Update 2.1 release notes with specifics about master/manager renaming.

ctubbsii commented on _docs-2/administration/upgrading.md:
```diff
@@ -4,6 +4,33 @@ category: administration
+
+The recommended way to upgrade from a prior 2.0.x release is to stop Accumulo, upgrade to 2.1 and
+then start 2.1. To upgrade from a release prior to 2.0, follow the [below steps](#upgrading-from-189-to-20)
+to upgrade to 2.0 and then perform the upgrade to 2.1. A direct upgrade from releases prior to 2.0
+has not been tested.
```
We need to support upgrade from 1.10, since 1.10 is an LTM release and that is a supported upgrade path. So, upgrading from 1.10 should definitely be tested before 2.1 is released.

brianloss replied:
Yeah, that makes sense. I'll update the comment.

ctubbsii commented on _posts/release/2020-01-19-accumulo-2.1.0.md:
```diff
@@ -1,10 +1,20 @@
 ---
 title: Apache Accumulo 2.1.0
+sortableversion: '02.01.00'
+archived_critical: true
```
```suggestion
sortableversion: '02.01.00'
```

Archiving it for critical bugs is a bit premature, given that it hasn't been released yet. :smiley_cat:

ctubbsii reviewed (commented):
For the most part, looks good to me. However, we won't be able to merge this and update the 2.x docs until after 2.1 is released.

brianloss commented:
> For the most part, looks good to me. However, we won't be able to merge this and update the 2.x docs until after 2.1 is released.

@ctubbsii sure, I just wanted to have the docs done. Do you think I should mark this PR as a draft, then? Or do we have some better way to leave it standing until the 2.1 release?

ctubbsii reviewed (approved):
> Do you think I should mark this PR as a draft, then? Or do we have some better way to leave it standing until the 2.1 release?

I think the best practice is to just leave it sit for now. We can revisit when we update the site for the 2.1 release. I think there may be another PR in that status as well. It doesn't need to be Draft or anything like that.

ctubbsii commented:
@brianloss I created a new base branch to prep changes for the next release, so we can get them merged, and keep them rebased onto main. This one, and #232 are both in that situation at the moment.

ctubbsii commented:
I merged this and #232 into the next-release branch, extracted the pieces that could be added to the main branch now in commit b7ce44494f9d39ccf9df34cc76a660a6b31630ac, and rebased the next-release branch onto that up-to-date main, and created a new PR to eventually merge that branch into main in #265

brianloss commented:
> I merged this and #232 into the next-release branch, extracted the pieces that could be added to the main branch now in commit [b7ce444](https://github.com/apache/accumulo-website/commit/b7ce44494f9d39ccf9df34cc76a660a6b31630ac), and rebased the next-release branch onto that up-to-date main, and created a new PR to eventually merge that branch into main in #265

Thanks for taking care of this!
